src/views/Home.jsx

```diff
@@ -1,11 +1,16 @@
 import './Home.css';
 
-export function Home() {
+export function Home(props) {
+	const { onClick } = props;
```

Comment:
If we wanted, we would destructure the props in the function declaration:

`export function Home({ onClick }) {};`

Reply:
Thank you for this!